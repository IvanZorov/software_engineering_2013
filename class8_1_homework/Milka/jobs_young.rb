require 'csv'

array = Array.new()
CSV.foreach(ARGV[0]) do |row|
	if row[1].to_i <= 30
		array << row
	end
end
array.sort!{|a,b| b[3].to_f <=> a[3].to_f}
CSV.open("jobs_young_result.csv","w") do |csv|
	array.each do |element|
	csv << element
	end
end


