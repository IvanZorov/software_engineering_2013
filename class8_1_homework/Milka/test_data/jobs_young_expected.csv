Chadwick Agular,20,taxi driver,20005.43
Hillary Mona,25,cook,7089.00
Elicia Resendez,19,director,6570.75
Boris Delafuente,23,programer,6400.50
Glenda Endo,23,programer,5499.24
Samella Tufts,27,programer,2410.00
Kellie Paulin,23,programer,2400.00
Leota Alvares,25,cook,2359.35
Robbi Liang,27,programer,1674.35
Chastity Budd,25,programer,1585.00
Meagan Riddle,25,sport,1553.15
Ouida Brice,18,director,876.00
Sade Shatley,21,director,20.50
